less/Home/Home.less

```diff
@@ -0,0 +1,291 @@
+                    margin-right: 5px;
+                    margin-bottom: 30px;
+                }
+            }
+            .content {
+                h2 {
+                    font-size: 2.8rem;
+                    font-family: Roboto, Arial, sans-serif;
+                    color: #222222;
+                    font-weight: bold;
+                    margin-bottom: 30px;
+                    margin-left: 20px;
+                    @media @mobile {
+                        width: 90%;
+                        font-size: 1.7rem;
+                        margin-left: 20px;
+                    }
+                }
+                p {
+                    margin-bottom: 30px;
+                    margin-left: 20px;
+                    width: 450px;
+                    font-size: 1.6rem;
+                    line-height: 24px;
+                    font-family: Roboto, Arial, sans-serif;
+                    color: #222222;
+                    @media @mobile {
+                        width: 95%;
+                        margin: 20px;
+                        line-height: 24px;
+                    }
+                }
+                .button {
+                    margin-left: 20px;
+                    .button-creator(25%, 1.4rem);
+                    padding: 12px;
+                    @media @mobile {
+                        margin-top: 20px;
+                        margin-left: 30px;
+                        // margin-left: 50px
+                        margin-bottom: 20%;
+                        .button-creator(80%, 1.4rem);
+                    }
+                }
+            }
+        }
+        .recent-projects {
+            .flex(@jc: center, @ai: center, @dir: column);
+            .width-100;
+            @media @largedst {
+                // width: 90%;
+            }
+            h2 {
+                align-self: flex-start;
+                font-size: @font-sub-heading;
+                font-family: Roboto, Arial, sans-serif;
+                color: #222222;
+                font-weight: bold;
+                margin-bottom: 30px;
+            }
+            .villa-heading {
+                position: relative;
+                .width-100;
+                .img-villas-mobile {
+                    display: none;
+                }
+                @media @mobile {
+                    .img-villas {
+                        display: none;
+                    }
+                    .img-villas-mobile {
+                        display: block;
+                        width: 100%;
+                        // width:95%;
+                    }
+                }
+                h3 {
+                    .img-heading(@right: 25px, @bottom: -25px, @width: 250px);
+                    // @right:-65px;
+                    @media @mobile {
+                        .img-heading(@right: 34px, @bottom: -65px, @width: 250px);
+                    }
+                }
+            }
+            .villas-content {
```

Comment:
try and have a single class for all these similar sections, and then have differentiating styles for only the aspects in which they're different. Otherwise this will be very repetitive and if you fix something in one spot it won't copy over to the other areas.